### ISSUE TITLE
HLAx without docker

### ISSUE THREAD

**Issue by Rachelmarty20** [humanlongevity/HLA#7]
I am interested in using HLAx in my research; however, I am unable to use docker on my cluster for security reasons. Can you provide alternative instructions for running HLAx without docker?

Thanks!

**Comment by tanghaibao** [comment]
@Rachelmarty20 I don't yet have time to make an automated build script yet. However, below is some information that might get you started.

While I understand that you don’t plan to use docker, the installation commands in Github would still be helpful as “step-by-step” instructions.

https://github.com/humanlongevity/HLA/blob/master/docker/dev.dockerfile
https://github.com/humanlongevity/HLA/blob/master/docker/deploy.dockerfile

In short, you’ll need to install SAMTOOLS, BWA, SAMBAMBA, BEDTOOLS, several R libraries, and DIAMOND, as dependencies.

Happy to answer further questions you have with regards to installation and evaluation.

Haibao

**Comment by Rachelmarty20** [comment]
Wonderful. Thank you @tanghaibao
